scripts/check-signed.py

```diff
@@ -13,6 +13,9 @@ def _check_signed() -> bool:
     if not github_actor:
         print("Missing GitHub Actor!")
         return False
+    if github_actor == "dependabot":
```

Comment:
```suggestion
    if github_actor == "dependabot[bot]":
```

Reply:
For future references: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#responding-to-events